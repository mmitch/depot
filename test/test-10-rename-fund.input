# define funds

+FUND fund_A
+FUND fund_B

# transactions per day

@@ 18.12.2020

fund_A	+1,978 = 32,5    ! 1,21
fund_B	+0,316 = 35,72   ! 1,01

+RENAME fund_A fund_Z

@@ 12.01.2021

fund_Z	+1,722 = 32,5    ! 1,21
