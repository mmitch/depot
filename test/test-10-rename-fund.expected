.---------------------------------------------------------------------------------------------------------.
| ETF    | Anteile      | Wert         | Kurs        (seit Start) | Gewinn        (relativ ) | Stand      |
+--------+--------------+--------------+--------------------------+--------------------------+------------+
| fund_B |    0.316 St. |    35.72 EUR |  113.038 EUR  (  +0.00%) |    -1.01 EUR  (  -2.75%) | 18.12.2020 |
| fund_Z |    3.700 St. |    69.83 EUR |   18.873 EUR  ( +14.87%) |     2.41 EUR  (  +3.58%) | 12.01.2021 |
+--------+--------------+--------------+--------------------------+--------------------------+------------+
| Depot  |              |   105.55 EUR |                          |     1.40 EUR  (  +1.35%) |            |
'--------+--------------+--------------+--------------------------+--------------------------+------------'
